docs/website/contents/for-developers/QueryVersioning.md

```diff
@@ -57,11 +77,21 @@ The supported query versions are only enforced in the [Shelley query encoder][sh
 
  3. Depending on the previous step:
 
-     - if necessary, add a new [`CardanoNodeToClientVersion`][cardano-n2c] and adapt [`supportedNodeToClientVersions`][supportedNodeToClientVersions] (as well as Shelley's [`supportedNodeToClientVersions`][shelley-supportedNodeToClientVersions], which is only used in tests)
+     - if necessary, add a new [`CardanoNodeToClientVersion`][cardano-n2c] and adapt [`supportedNodeToClientVersions`][supportedNodeToClientVersions] (as well as Shelley's [`supportedNodeToClientVersions`][shelley-supportedNodeToClientVersions], which is only used in tests).
+     We use this mapping  because `SupportedNetworkProtocolVersion` is not a composable instance.
 
      - if necessary, add a new [`ShelleyNodeToClientVersion`][shelley-n2c] and adapt [`querySupportedVersion`][query-supported-version]
 
- 4. In many cases, these changes will make our golden tests fail (due to missing files); to fix CI, run the tests locally and check in the newly generated files.
+4. In case the network node to client versions (eg `NodeToClient_V16`) is not linked to to the `CardanoNodeToClientVersion` mentioned above, we need to do this by adding an extra entry to `supportedNodeToClientVersions`, in [Ouroboros.Consensus.Cardano.Node](https://github.com/input-output-hk/ouroboros-consensus/blob/main/ouroboros-consensus-cardano/src/ouroboros-consensus-cardano/Ouroboros/Consensus/Cardano/Node.hs).
```

Comment:
Very minor, as this is eg how you abbreviate it above:
```suggestion
4. In case the network node-to-client versions (eg `NodeToClient_V16`) is not linked to to the `CardanoNodeToClientVersion` mentioned above, we need to do this by adding an extra entry to `supportedNodeToClientVersions`, in [Ouroboros.Consensus.Cardano.Node](https://github.com/input-output-hk/ouroboros-consensus/blob/main/ouroboros-consensus-cardano/src/ouroboros-consensus-cardano/Ouroboros/Consensus/Cardano/Node.hs).
```